luaui/Widgets/Shaders/deferred_lights_gl4.vert.glsl

```diff
@@ -194,7 +194,14 @@ void main()
 				else {
 					colormod =  cos(elapsedframes * 6.2831853 * colortime ) * 0.5 + 0.5;
 				}
-				v_lightcolor.rgb = mix(v_lightcolor.rgb, worldposrad2.rgb, colormod); 
+				v_lightcolor.rgb = mix(v_lightcolor.rgb, color2.rgb, colormod); 
+			}
+			if (worldposrad2.w < 1.0) {
+				lightCenterPosition += timeInfo.w * worldposrad2.xyz;
+			}else{
+				// Note: worldposrad2.w is an excellent place to add orbit-style world-placement light animations
+				vec3 lightWorldMovement = sin(time * 0.017453292 * worldposrad2.xyz) * worldposrad2.w;
```

Comment:
why would "time" have a degree-to-radians conversion?

Reply:
orbit period